.github/workflows/build.yaml

```diff
@@ -6,7 +6,6 @@ on:
 
 jobs:
   build-artifacts-ARM64-macOS:
-    concurrency: build-ARM64-macOS
```

Comment:
?

Reply:
I think I got rid of this so it'd build faster?

Reply:
Would be nice to know what the problem was so we don't toggle between having it and not having it until the end of time...